### DISCUSSION TITLE
Older Modern Fortran Version

### DISCUSSION THREAD

**Discussion by jbiscan-dot** [krvajal/vscode-fortran-support#980] [Q&A - deprecated see Fortran Discourse]
Hello,

Is there a way I can obtain older versions of Modern Fortran that are compatible with older VS code versions?

**Comment by gnikit** [comment]
This is somewhat possible, but I **must urge against it**. Modern Fortran supports VS Code versions up to 1.63+ which was released in November of 2021. You might be able to roll back to an older version of the extension with an even previous minimum VS Code version requirement by following the steps in [this](https://stackoverflow.com/questions/42626065/vs-code-how-to-rollback-extension-install-specific-extension-version) StackOverflow post, but I think a lot of the old versions got scrapped by Microsoft when we moved the extension to Fortran-lang.
